feature/bgp/tests/local_bgp_test/local_bgp_test.go

```diff
@@ -288,7 +335,7 @@ func TestParameters(t *testing.T) {
 				Timers: &oc.NetworkInstance_Protocol_Bgp_Neighbor_Timers{
 					HoldTime:           ygot.Uint16(holdTime),
 					NegotiatedHoldTime: ygot.Uint16(135),
-					KeepaliveInterval:  ygot.Uint16(keepAlive),
+					KeepaliveInterval:  ygot.Uint16(45),
```

Comment:
Shouldn't dutConf.Timers.KeepaliveInterval be the same as wantState.Timers.KeepaliveInterval ?  I don't think it is correct to just change this one value without also changing the other.